idaes/core/scaling/tests/test_custom_scaler_base.py

```diff
@@ -608,15 +608,12 @@ def test_call_submodel_scaler_method_no_scaler(self, caplog):
         m.b = Block([1, 2, 3])
 
         sb = CustomScalerBase()
-        sb.call_submodel_scaler_method(m, "b", method="dummy_method", overwrite=True)
+        sb.call_submodel_scaler_method(m.b, method="dummy_method", overwrite=True)
 
         for bd in m.b.values():
             assert not hasattr(bd, "_dummy_scaler_test")
 
-        assert (
-            "No default Scaler set for unknown.b. Cannot call dummy_method."
-            in caplog.text
-        )
+        assert "No default Scaler set for b. Cannot call dummy_method." in caplog.text
```

Comment:
I wonder if only referencing dummy_method in error message may confuse maintainer in the future

Reply:
Do you have a suggestion for a better message?

Reply:
The message is generated dynamically using the name of the method the user asked for. In this test case, the method I asked for was named "dummy_method", but a user would see the name of the method they specified.

Reply:
Then I think it's fine